src/helpers/messageHelpers.ts

```diff
@@ -58,20 +66,27 @@ export async function getThreadAuthor(channel: ThreadChannel): Promise<User | un
 	const firstMessage = await getFirstMessageInChannel(channel);
 	const author = firstMessage?.mentions.users.first();
 
-	if (!author) console.log(`Could not determine author of thread "${channel.name}"`);
+	if (!author)
+		console.log(`Could not determine author of thread "${channel.name}"`);
```

Comment:
```suggestion

```
Indentation like this looks so bad that I'd rather remove it completely than keep it 😄 (Pretty pointless log anyways)

Reply:
I've fixed this indentation issue now, do you still want to remove this line though?

Reply:
Let's remove it, yeah :)